src/i18n/locales/en/translation.json

```
@@ -259,6 +261,16 @@
     "sweep_amount_breakdown_frozen_balance": "Frozen or locked balance",
     "sweep_amount_breakdown_estimated_amount": "Estimated amount to be sent",
     "sweep_amount_breakdown_explanation": "A sweep transaction will consume all UTXOs of a mixdepth leaving no coins behind except those that have been <1>frozen</1> or <3>time-locked</3>. Onchain transaction fees and market maker fees will be deducted from the amount so as to leave zero change. The exact transaction amount can only be calculated by JoinMarket at the point when the transaction is made. Therefore the estimated amount shown might deviate from the actually sent amount. Refer to the <5>JoinMarket documentation</5> for more details.",
+    "fee_breakdown": {
+      "absolute_limit": "Absolute fee limit (<balance/>) * {{ num }} collaborators",
+      "or_relative_limit": "OR relative fee limit ({{ percentage }}) * {{ num }} collaborators",
+      "not_set": "Not set",
+      "too_low": "Too low",
+      "total_estimate": "Max collaborators fee",
+      "plus_mining_fee": "Plus mining fees",
+      "why_cant_estimate_mining_fee": "Why can't we estimate mining fees?",
+      "cant_estimate_mining_fee_info": "The amount of inputs and outputs is not know beforehand, so it is not possible to know the transaction size, hence the transaction fees cannot be estimated.<br>In addition to this, if you have set a \"Block target\" as mining fee (read: \"Include my tx in the next n blocks\", instead of a sats/vByte value), we cannot estimate the fee needed for your transaction since Jam does not have access to the current mempool conditions."
```

```suggestion
      "cant_estimate_mining_fee_info": "The amount of inputs and outputs is not know beforehand, so it is not possible to know the transaction size, hence the transaction fees cannot be estimated.<br>In addition to this, if you have set a \"Block target\" as mining fee (read: \"Include my tx in the next n blocks\", instead of a sats/vByte value), the fee needed for your transaction cannot be calculated since Jam does not have access to the current mempool conditions."
```

:+1: for the text and explanation in general. Nice!
Small nit: I'd suggest to not use "we", but rather passive voice. Okay for you?